apps/web/src/features/walletconnect/components/WalletConnectProvider/index.tsx

```diff
@@ -73,6 +75,7 @@ export const WalletConnectProvider = ({ children }: { children: ReactNode }) =>
       const { topic } = event
       const session = walletConnect.getActiveSessions().find((s) => s.topic === topic)
       const requestChainId = stripEip155Prefix(event.params.chainId)
+      const peerName = (session && getPeerName(session.peer)) || FALLBACK_PEER_NAME
```

Comment:
nit: `FALLBACK_PEER_NAME` could be moved into `getPeerName`

Reply:
No, there are places where the fallback is changed to "Unknown dApp".